README.md

```diff
@@ -13,6 +13,7 @@ Here is the list of device backends that we support:
 - [RNG](https://github.com/rust-vmm/vhost-device/blob/main/crates/vhost-device-rng/README.md)
 - [SCMI](https://github.com/rust-vmm/vhost-device/blob/main/crates/vhost-device-scmi/README.md)
 - [SCSI](https://github.com/rust-vmm/vhost-device/blob/main/crates/vhost-device-scsi/README.md)
+- [VIDEO](https://github.com/rust-vmm/vhost-device/blob/main/staging/crates/vhost-device-video/README.md)
```

Comment:
Oops, left the crates folder here. I comment as a reminder.

Reply:
But I can use the occasion to ask: do we want the staging crates listed separately in the main README?

Reply:
This is also a good point for #466.

IMHO we should list staging crates in the main README, I think @epilys is going to update that PR adding a section there where we can also move this link.